quil-rs/src/program/calibration.rs

```diff
@@ -298,22 +502,23 @@ impl Calibrations {
 
         for calibration in self.iter_calibrations() {
             // Filter out non-matching calibrations: check rules 1-4
-            if calibration.name != gate.name
-                || calibration.modifiers != gate.modifiers
-                || calibration.parameters.len() != gate.parameters.len()
-                || calibration.qubits.len() != gate.qubits.len()
+            if calibration.identifier.name != gate.name
+                || calibration.identifier.modifiers != gate.modifiers
+                || calibration.identifier.parameters.len() != gate.parameters.len()
+                || calibration.identifier.qubits.len() != gate.qubits.len()
```

Comment:
Would it make sense to throw this in an `is_match(&self, gate: &Gate)` method on CalibrationIdentifier?

Reply:
yep, good thought, done